substratevm/mx.substratevm/suite.py

```diff
@@ -683,6 +683,24 @@
             "spotbugs": "false",
         },
 
+        "com.oracle.svm.test.jdk17": {
+            "subDir": "src",
+            "sourceDirs": ["src"],
+            "dependencies": [
+                "mx:JUNIT_TOOL",
+                "sdk:GRAAL_SDK",
+                "SVM",
+            ],
+            "checkstyle": "com.oracle.svm.test.jdk17",
+            "checkstyleVersion" : "8.36.1",
+            "workingSets": "SVM",
+            "annotationProcessors": [
+                "compiler:GRAAL_PROCESSOR",
+            ],
+            "javaCompliance": "17+",
+            "spotbugs": "false",
```

Comment:
```suggestion
            "spotbugs": "false",
            "testProject": True,
```

You need this to get the CI happy :)

Reply:
Thanks, done!